src/main/java/frc/robot/Constants.java

```diff
@@ -56,6 +56,8 @@ public static final class FeatureFlags {
         public static final boolean runScoring = false;
```

Comment:
runScoring will need to be true to see the encoder (just a note for tomorrow's testing)